webrtc-stats.html

```diff
@@ -1269,6 +1271,41 @@
                 </p>
               </dd>
               <dt>
+                <dfn><code>playoutTimestamp</code></dfn> of type <span class="idlMemberType"><a>
+                DOMHighResTimeStamp</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only valid for remote sources. This is the estimated playout time of this track.
+                  The playout time is the NTP timestamp of the last played out audio sample or video
+                  frame that has a known timestamp, extrapolated with the time elapsed since it was
+                  played out. This is the "current time" of the track in NTP clock time of the
+                  sender and can be present even if there is no audio or video currently playing.
```

Comment:
Would the AVSync still be estimated if the clockrates for audio and video are different?

Perhaps some explanation for calculating AV Sync would be helpful.